### ISSUE TITLE
Search using opponent name

### ISSUE THREAD

**Issue by maharjanaman** [QuickBlox/quickblox-react-native-sdk#71]
Is there any option to search with opponent name? Current search is happening only with chat name causing one opponent to be able to search and other is unable to search.

**Comment by IlliaChemolosov** [comment]
@maharjanaman 
You can find comprehensive information about operators and fields available for searching dialogs in our documentation at https://docs.quickblox.com/docs/react-native-chat-dialogs#search-operators.

**Comment by maharjanaman** [comment]
Thanks for the response.
From the docs, the field `name` searches the dialog by chat name but not the opponent name for a private chat. Is this the case? If yes, is there any way to search by opponent name for the private chat?

**Comment by IlliaChemolosov** [comment]
@maharjanaman Yes, in the case of private chats 'name' field, the search primarily looks for the opponent's name. You can also utilize the 'all' operator, the 'occupant_ids' key, and the user ID to specifically search for a dialogue using the opponent's name in a private chat.

**Comment by maharjanaman** [comment]
@IlliaChemolosov opponent's name is not being used while using the 'name' field for search. This is on the latest version of react native sdk.
